npm/ng-packs/packages/core/src/lib/abstracts/auth-error-filter.ts

```diff
@@ -0,0 +1,34 @@
+import { signal } from '@angular/core';
+import { AuthErrorEvent, AuthErrorFilter } from '../models';
+
+export abstract class AbstractAuthErrorFilter<T, E> {
+  readonly #filters = signal<Array<T>>([]);
+  filters = this.#filters.asReadonly();
```

Comment:
We just define the readonly Signal. We shouldn't define private object in abstract